mlflow/ml-package-versions.yml

```diff
@@ -69,64 +69,28 @@ tensorflow:
     minimum: "2.3.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       # Requirements to run tests for keras
-      "== dev": ["scikit-learn", "pyspark", "pyarrow"]
+      ">= 0.0.0": ["scikit-learn", "pyspark", "pyarrow"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py
-
-      # Run tests for keras against keras-nightly that's installed with tf-nightly:
-      # https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/tools/pip_package/setup.py#L110-L122
-      if [ "$PACKAGE_VERSION" == "dev" ]; then
-        pytest tests/keras/test_keras_model_export.py
-      fi
+      tf_version=$(pip freeze | grep 'tensorflow==' | grep -Eo '[0-9].[0-9]')
+      pip install tensorflow-estimator==${tf_version}
```

Comment:
Do we need to reinstall `tensorflow-estimator`?

Reply:
This line ends with the following message in all the tf cross version tests:

```
Requirement already satisfied: tensorflow-estimator...
```

Reply:
I will remove it.